sdk/storage/storage-blob/src/ContainerClient.ts

```diff
@@ -1558,10 +1558,10 @@ export class ContainerClient extends StorageClient {
       let urlWithoutSAS = this.url.split("?")[0]; // removing the sas part of url if present
```

Comment:
The URL could be in emulator format, e.g. [https://127.0.0.1/accountName/containerName](https://127.0.0.1/accountName/containerName), If it's by design to be supported, consider to create a github issue for further optimization.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/5446